pkg/controlplane/authz/manager.go

```diff
@@ -40,6 +40,10 @@ const (
 	// the number of seconds a JWT access token is valid before it expires.
 	jwtExpirySeconds = 5
 
+	ClientNameLabel       = "clusterlink/metadata.clientName"
+	ClientNamespaceLabel  = "clusterlink/metadata.clientNamespace"
+	ClientSALabel         = "clusterlink/metadata.clientServiceAccount"
+	ClientLabelsPrefix    = "client/metadata."
```

Comment:
suggest being explicit that these are labels/tags (i.e., `metadata.label.*`)?
Also, unsure why the designation changes to `client` here. Why separate client from server and do we need a new "grouping" for attributes (besides `clusterlink`)? If so, consider using `k8s` for the relevant attributes (namespace, serviceaccount, labels, ...) instead of clusterlink and reserve that to attributes relating to CL objects (e.g., peer, import, export, ...)